pkg/tagset/tags_cache.go

```diff
@@ -0,0 +1,106 @@
+package tagset
+
+// tagsCache caches Tags instances using purpose-specific cache keys.
+//
+// Note that tagsCache instances are not threadsafe
+type tagsCache struct {
+	// number of inserts between rotations
+	insertsPerRotation int
+
+	// number of inserts remaining before next rotation
+	untilRotate int
+
+	// tagests contains the constitutent tagset maps. This is a slice of length
+	// cacheCount.  The first map is the newest, into which new values will be
```

Comment:
I know why you're doing this two spaces thing but I can't help my eyes to not catch it as a typo 😅

Reply:
Sorry!  `sed -i -e 's/\.  /. /g' $(git grep -l '\.  ')` fixes it up..

Reply:
it was a silly remark, please ignore ofc.